test/gui/shared/scripts/helpers/api/oc10.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import helpers.api.HttpHelper as request
+import helpers.api.Provisioning as provisioning
```

Comment:
this cyclic import might cause problem later. if we can have separate helper file for common functions then would be nice

Reply:
just placed `assertHttpStatus` in HttpHelper to avoid cyclic import, is it okay?